<launch>
	<rosparam file="$(find ndt2_control)/config/config.yaml"  />
	<node name="voliro_controller" pkg="ndt2_control" type="voliro_controller" output="screen" />
</launch>
